pkg/test/constants/presubmitconstants.go

```diff
@@ -59,6 +59,7 @@ var (
 		"iam":                  {"oidcworkloadidentitypoolprovider"},
 		"iap":                  {"iapidentityawareproxyclient"},
 		"identityplatform":     {"identityplatformoauthidpconfig"},
+		"ids":                  {"idsendpoint"},
```

Comment:
The key (lowercase service name) is `cloudids` I think? Or do you plan to update the service to `ids`?

Reply:
Good call! I forgot about the top level dir.